client/extensions/woocommerce/state/data-layer/request/test/index.js

```diff
@@ -0,0 +1,223 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import { spy } from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import { get, post, put, del } from '../actions';
+import { handleRequest } from '../';
+import useNock from 'test/helpers/use-nock';
+import { useSandbox } from 'test/helpers/use-sinon';
+import {
+	WOOCOMMERCE_ERROR_SET,
+	WOOCOMMERCE_REQUEST_SUCCESS,
+	WOOCOMMERCE_REQUEST_FAILURE,
+} from 'woocommerce/state/action-types';
+
+describe( 'handlers', () => {
+	const siteId = 123;
+
+	describe( '#get', () => {
+		const getResponse = { name: 'bogus get response', bogus: true };
+
+		useSandbox();
+		useNock( ( nock ) => {
+			nock( 'https://public-api.wordpress.com:443' )
+				.get( '/rest/v1.1/jetpack-blogs/123/rest-api/' )
+				.query( { path: '/wc/v3/bogus_endpoint&_method=get', json: true } )
+				.reply( 200, { data: getResponse } );
+		} );
+
+		it( 'should handle get success', () => {
+			const store = {
+				dispatch: spy(),
+			};
+
+			const action = get( siteId, 'bogus_endpoint' );
```

Comment:
I just wanted to denote that it's not a "real" endpoint because I didn't want these tests to be tied to any specific API endpoints. Maybe "placeholder" is a better term?